lib/quicksilver-config-client.js

```diff
@@ -51,6 +53,21 @@ class QuicksilverConfigClient {
   getExchangeSettings() {
     return true;
   }
+
+  /**
+   * Check that configuration file define in .env as QS_CONFIG_PATH is accessable.
+   */
+  configFileFound() {
+    return new Promise((resolve) => {
+      fs.access(this.qsConfigPath, fs.F_OK, (err) => {
+        const fileFound = true;
+        if (err) {
+          throw new Error(err);
+        }
+        resolve(fileFound);
```

Comment:
You can just resolve it for true here, you don't need to declare a constant for `true` above.